space-cli/cmd/modules/database/helpers.go

```diff
@@ -0,0 +1,132 @@
+package database
+
+import (
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils"
+	"github.com/spf13/cobra"
+)
+
+func validDBRulesArgsFunc(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
```suggestion
func dbRulesAutoCompleteFun(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```
do the same with other functions in this file